src/applications/ivc-champva/shared/components/fileUploads/upload.js

```diff
@@ -67,7 +67,12 @@ export const fileUploadUi = content => {
       },
     }),
     // hideOnReview: true,
-    // attachmentName: true,
+    // attachmentName: false,
+    attachmentName: content?.attachmentName
+      ? {
+          'ui:title': 'Document name',
+        }
+      : false,
```

Comment:
lets us pass the `attachmentName` prop to toggle showing the editable filename text field while maintaining backward compatibility with other forms' usage of this upload component.